draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -239,31 +237,50 @@ The detailed algorithm is described in (#verifier_verification).
 
 This section defines data formats for SD-JWTs, Disclosures, Holder Binding JWTs and formats for combining these elements for transport.
 
-## Format of an SD-JWT
+## Format of an SD-JWT {#creating_sd_jwt}
 
-An SD-JWT is a JWT that MUST be signed using the Issuer's private key. The
-payload of an SD-JWT MUST contain the `_sd_alg` claim
-described in (#hash_function_claim). The SD-JWT payload MAY contain one or more selectively disclosable claims. It MAY also contain a Holder's public key or a reference
-thereto, as well as further claims such as `iss`, `iat`, etc. as defined or
-required by the application using SD-JWTs.
+An SD-JWT is a JWT that MUST be signed using the Issuer's private key.
+It MUST use a JWS asymmetric digital signature algorithm. It
+MUST NOT use `none` or an identifier for a symmetric algorithm (MAC).
+
+The payload of an SD-JWT is an object according to the following rules:
```

Comment:
```suggestion
The payload of an SD-JWT is a JSON object according to the following rules:
```

I know it's implied and maybe even obvious but I still think explicitly saying "JSON object" here is worthwhile.